src/simpla-element-behavior.html

```diff
@@ -10,185 +10,218 @@
 
   window.SimplaBehaviors = window.SimplaBehaviors || {};
 
-  SimplaBehaviors.Element = {
-    properties: {
+  SimplaBehaviors.Element = function(config) {
+    let {
+          type,
+          getCallback,
+          setCallback,
+          dataProperties,
+          updateOn
```

Comment:
In that case I'd still much rather use deep observation of the properties. It's just a much cleaner interface - "here's my data property, update Simpla whenever it changes" - it shouldn't matter if that change is in a simple or complex value.

Re: simpla-text, regardless I think value should be a property rather than a getter, it's just more robust (from an interface point of view)